tests/apps/migrations/commands/test_forwards_backwards.py

```diff
@@ -174,6 +174,8 @@ def test_forwards_fake(self):
                 "2020-12-17T18:44:30",
                 "2020-12-17T18:44:39",
                 "2020-12-17T18:44:44",
+                "2021-07-19T12:40:31:140762",
+                "2021-07-19T13:34:56:395354",
```

Comment:
Presumably these new migrations are to test creating tables with the new metaclass behaviour?

Reply:
Shamefully I don't know why they were automatically generated when I ran the tests. I guess because I added new test tables?

Reply:
No worries - they were probably just added by mistake when a file got moved. I've removed them, and added a new migration file, as I just realised some of the tables in the example app weren't covered by migrations.